lib/init.js

```diff
@@ -8,7 +8,9 @@ function init (hoodie) {
   account.on('signout', function () {
     // TODO: prevent data loss on sign out:
     //       https://github.com/hoodiehq/hoodie-client/issues/22
-    store.clear()
+    store.reset({
+      name: hoodie.account.id
+    })
```

Comment:
we need to pass in hoodie.account.id on `signin` as well, as the `hoodie.account.id` changes to the one of the account you sign into